utils/metered_cache.go

```diff
@@ -54,6 +57,7 @@ func NewMeteredCache(size int, journal string, namespace string, updateFrequency
 		mc.gets = metrics.GetOrRegisterGauge(fmt.Sprintf("%s/gets", namespace), nil)
 		mc.sets = metrics.GetOrRegisterGauge(fmt.Sprintf("%s/sets", namespace), nil)
 		mc.misses = metrics.GetOrRegisterGauge(fmt.Sprintf("%s/misses", namespace), nil)
+		mc.statsTime = metrics.GetOrRegisterGauge(fmt.Sprintf("%s/statsTime", namespace), nil)
```

Comment:
I think this is should be `RegisteredResettingTimer`?

Reply:
Reverted that change since using a timer would be more suited for individual measurement. We can use gauge or meter for the cumulative time